Dockerfile

```diff
@@ -10,11 +10,15 @@ RUN useradd $username -u $user_id --create-home --user-group
 
 COPY ./dist/suzieq-$version-py3-none-any.whl  /tmp/
```

Comment:
```suggestion
```
If the wheel changes we invalidate all the following layers, better to put it some lines after (already included in the previous comment)